test/build.gradle

```diff
@@ -50,7 +50,7 @@ android {
     productFlavors {
         prodTesting {
             manifestPlaceholders.in_app_messaging_sub_id = property("IAM_SUBSCRIPTION_KEY") ?: ""
-            manifestPlaceholders.config_url = property("CONFIG_URL") ?: ""
+            manifestPlaceholders.config_url = property("CONFIG_URL_TEST_PROD") ?: ""
```

Comment:
is `CONFIG_URL_TEST_PROD` just for integration tests?

Reply:
`CONFIG_URL_TEST_PROD` this is used for integration test, unit test, and sample app